node/cmd/calico/cni_conf_test.go

```diff
@@ -23,13 +23,17 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("CNI config template tests", func() {
+var _ = Describe("Windows CNI config template tests", func() {
```

Comment:
huh, I know this isn't added in your PR but I'm not sure why this test file is in the cmd package - it's not testing the cmd package functionality right?

Reply:
Yeah you're right, though I can sort of see why they were added here. Let me find a better place.

Reply:
I've moved them into the a subdir inside `windows-packaging`: 4f9376c1f0d0ae41e5cf2ad21814c3fa219f6190

I checked that the tests are not included in the calico-windows.zip file